leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/LeshanClient.java

```diff
@@ -211,6 +214,13 @@ public void start() {
         LOG.info("Starting Leshan client ...");
         endpointsManager.start();
         engine.start();
+
+        for (LwM2mObjectEnabler objectEnabler : objectTree.getObjectEnablers().values()) {
+            if (objectEnabler instanceof Startable) {
+                ((Startable) objectEnabler).start();
+            }
+        }
```

Comment:
An idea is to add a start/stop/destroy method to `LwM2mObjectEnabler` and move the loop in this class ?
(This is maybe in term of separation of concern)

Reply:
Thank you. That sounds good, I'll do that.

Reply:
@sbernard31 Do you mean `LwM2mObjectTree` instead of `LwM2mObjectEnabler` ?

Reply:
Uups, I wanted to  say `LwM2mObjectTree` :sweat_smile:

Reply:
haha, thanks for your advice! I did this at 603cb2a9d0afcd3c3c40c5190621b37e7422f04f